### DISCUSSION TITLE
Wireless serial devices for NMEA link (BT or WiFi) - which ESP8266 firmware

### DISCUSSION THREAD

**Discussion by j-lang** [Rodemfr/MicronetToNMEA#79] [Q&A] [answered]
Is there any advice, which firmware is working as a WiFi serial gateway for OpenCPN or similar application? 
What are the main points to get the inteface running.
Thx Hannes

**Comment by dwarning** [comment, marked as answer]
These both alternatives working with my Serial WiFi v1.0 board:

ESP8266_NMEA_BRIDGE.ino:
http://forward.com.au
// V1.1.0
// 2022/01/11 V1.1.0 added DHCP server to AccessPoint

nmea_bridge.ino:
https://github.com/alvra/nmea-bridge

Dietmar

**Reply by j-lang** [reply]
I could not find any code on the first link: http://forward.com.au/ 
but with code from https://github.com/alvra/nmea-bridge the wifi connection works within minutes, thank you. 

I have a samsung android smartphone, when the smartphone is connected to the nmea wifi access point, internet access via mobile network is disabled, so its not possible for a map application to download the map. Has anybody a solution for this problem.

On the other hand, I also have the possibility to connect the smartphone via bluetooth, but could not find a way to use the nmea info in Sailracer oder Navmonitor on  an android smartphone. Is there experience how a navigation app can receive nmea data via bluetooth? 
THX Hannes

**Reply by Rodemfr** [reply]
I use HC06 Bluetooth dongle for this exact reason : not loosing Internet. Not all apps can use bluetooth properly but I know two which work well BT : [Avnav](https://www.wellenvogel.net/software/avnav/docs/release.html?lang=en)(free) and [Sailgrib](https://www.sailgrib.com/)(non free). I use avnav with mbtile offline maps.
Another option is to use offline maps. I use [SAS Planet](https://sasplanet.geojamal.com/) to build mbtiles of marine maps. You can download navionics maps with *SAS Planet*.

**Reply by j-lang** [reply]
Than you.

**Comment by j-lang** [comment]
Great, thank you. I will test it within the next days.

**Comment by Rodemfr** [comment]
Transferring to discussions...

**Comment by Pacer27** [comment]
Hello,
here is tel link to the Down Under RS 232 Wifi Bridge
https://www.forward.com.au/pfod/CheapWifiShield/NMEAbridge/index.html
